assets/js/hooks/handle_timestamp_timezone.js

```diff
@@ -1,10 +1,10 @@
 function adaptUtcTimestampToUserTimezone(component) {
-    dateElement = document.getElementById(component.el.id)
-    utcTimestamp = dateElement.innerHTML
-    dateElement.innerHTML = new Date(utcTimestamp).toLocaleString().replace(",", "")
+    let dateElement = document.getElementById(component.el.id)
+    let utcTimestamp = dateElement.innerHTML.trim()
+    dateElement.innerHTML = new Date(utcTimestamp).toLocaleString('en-GB')
```

Comment:
do we really need these to be declared as `let` instead of `const` ?
Hint: in JS, `const` only means that the given variable cannot be reassigned to a different value. But setting fields / appending into it is possible:
```javascript
const myObj = {a: 1}
myObj = {a: 2} // this doesn't work as myObj is const
myObj.a = 2 // this works and sets the `a` field in myObj as 2
```

Reply:
However, I wonder if there is some sort of library that would enforce immutability in Js. Perhaps there is something like that for Typescript?

Reply:
I only know of this: 
https://immutable-js.com/

and the good ol' `Object.freeze`